airmozilla/popcorn/static/popcorn/js/editor.js

```
@@ -1,2 +1,29 @@
-// Initialize the editor with the div id and path to Popcorn Editor.
-PopcornEditor.init('editor', '/static/popcorn/PopcornEditor/editor.html');
+$(document).ready(function () {
+  function getSlug() {
+    var config = $('slug');
+    if (config.length) {
+      return config.data('url');
+    } else {
+      console.error('Slug not provided in DOM. Cannot get slug');
```

I would do `throw 'Slug not provided...'`